plugin.video.liveleak/default.py

```diff
@@ -1,196 +1,311 @@
-# Standard libraries
-import urllib, urllib2, urlparse, re
-from HTMLParser import HTMLParser
+# -*- coding: utf-8 -*-
+
+# Standard libraries - Python 2 & 3
+import re, requests, json
+from os import path
+from bs4 import BeautifulSoup as bs
+
+# Fallback if multiprocessing module is not available
+slow_mode = False
+try:
+    from multiprocessing.dummy import Pool
+except:
+    slow_mode = True
+
+try: # Python 3
```

Comment:
You could use `sys.version_info[0]` instead of blindingly detecting the python version trough a try/except